src/lib/src/datepicker/datepicker.component.spec.ts

```diff
@@ -85,4 +85,11 @@ describe('Component: DatepickerComponent', () => {
     expect(comp.date.getSeconds()).toEqual(seconds);
     expect(comp.date.getMilliseconds()).toEqual(milliseconds);
   });
+
+  it('should support displaying UTC date', () => {
+    comp.date = new Date(Date.UTC(2017, 0, 1, 0, 0, 0));
+    comp.UTC = true;
+    fix.detectChanges();
+    expect(comp.picker.getDate().getDate()).toEqual(1);
```

Comment:
Oh, I get it!  That 2nd getDate is just returning the day of the month.  C'mon javascript ... naming!